ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -1026,6 +1026,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8
+- as: aws-ipi-proxy-techpreview-po-p2-f14
+  cron: 53 8 16,29 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @proxy and @network-ovnkubernetes and not @fips'
+      TAG_VERSION: '@4.12'
+      TEST_FILTERS: ~CPaasrunOnly&;~MicroShiftOnly&;~NonPreRelease&;~VMonly&;~ProdrunOnly&;~StagerunOnly&;~DisconnectedOnly&
```

Comment:
@jianzhangbjz 
do not need to overwrite TEST_FILTERS and remove this line to take default value of the step.

it apply to all your test added.

Reply:
Why? What's the default `TEST_FILTERS`? And, where is it?

Reply:
for openshift-e2e-test-qe, it uses golang step openshift-extended-test.
for openshift-e2e-test-qe-destructive, it uses golang step openshift-extended-test-disruptive

the default value of TEST_FILTERS is in the definition of golang step.

the default value is for connnected cluster, so no need to overwrite it for connected cluster.
if your cluster is disconnected cluster, have to overwrite default value.

you could check existing test to find this.

Reply:
you will find the default value is not same with what I ever told you, for example `~CPaasrunOnly&` or `~VMonly&;`
it is due that such filter is hard-coded in script because such filters are always same for golang step whatever cluster it is. in order to reduce FILTER, set such filter in hard-coded. or else, when someone have to overwrite filters, it is long string.

Reply:
I see, thanks!